lib/loggerator/metrics.rb

```diff
@@ -1,18 +1,16 @@
-require_relative 'loggerator'
+require "loggerator"
 
 module Loggerator
   module Metrics
     include Loggerator
     extend self
 
-    @@metrics_name = 'loggerator'
-
     def name=(name)
-      @@metrics_name = name
+      Loggerator.config.metrics_app_name = name
     end
```

Comment:
If I remove this, I should probably also remove .`default_context=` from Log. Both would be breaking changes in the primary config pattern, which is why I didn't. Perhaps a deprication message, instead of outright removal at this point?